README.md

```diff
@@ -34,13 +34,14 @@ And then configure `package.json` to look like this:
 {
   "scripts": {
     "changelog": "uphold-scripts changelog $npm_package_version",
-    "lint": "uphold-scripts lint",
```

Comment:
This isn't working since our script automatically calls with a dot
```
#!/usr/bin/env sh

eslint --cache . $@
```

Source: https://github.com/uphold/uphold-scripts/blob/fix-lint-staged-readme/src/uphold-scripts-lint


However, this is a breaking change where every project should add a dot to the package json.

Reply:
We will release a minor (0.7.0).